assets/scripts/free-trial-wc-payments.js

```diff
@@ -27,17 +27,27 @@
 		} );
 	};
 
+	// Prefer wc.tracks.recordEvent since it supports debugging.
+	let recordEvent = null;
+	if ( window.wc && window.wc.tracks && window.wc.tracks.recordEvent ) {
+		recordEvent = window.wc.tracks.recordEvent;
+	} else if ( window.wcTracks && window.wcTracks.recordEvent ) {
+		recordEvent = window.wcTracks.recordEvent;
+	} else {
+		recordEvent = function () {};
+	}
```

Comment:
@ilyasfoo Is it possible to create a global function for this in window object so we don't need to have this in many places?

Reply:
@chihsuan  I was thinking that too but I'm not sure if I could guarantee a script that'll be loaded first across all uses. Since some of them will be loaded via PHP, some via external script.. Do you have any suggestions? 🤔

Reply:
I don't have suggestions now. Perhaps we can create a follow-up issue for it. 🤔

Reply:
Agreed, thanks!

Reply:
Follow-up issue created https://github.com/Automattic/wc-calypso-bridge/issues/1079